src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -175,9 +175,14 @@ private void populatePaymentGeneralLedgerPendingEntry(
             glPendingTransaction.setAccountNumber(paymentAccountDetail.getAccountNbr());
             glPendingTransaction.setSubAccountNumber(paymentAccountDetail.getSubAccountNbr());
             glPendingTransaction.setChartOfAccountsCode(paymentAccountDetail.getFinChartCode());
-
-            glPendingTransaction.setFinancialDocumentTypeCode(paymentAccountDetail.getPaymentDetail().getFinancialDocumentTypeCode());
-
+            
+            if (StringUtils.equals(paymentGroup.getDisbursementType().getCode(), PdpConstants.DisbursementTypeCodes.ACH)) {
+                glPendingTransaction.setFinancialDocumentTypeCode(state.documentTypeForDisbursementType(PdpConstants.DisbursementTypeCodes.ACH));
```

Comment:
The previous set functions were
glPendingTransaction.setFinancialDocumentTypeCode(achFdocTypeCode);
or
 glPendingTransaction.setFinancialDocumentTypeCode(checkFdocTypeCod);

I am not sure if achFdocTypeCode or checkFdocTypeCod  exist any longer, but you could use the PDP constant.
Why do you need to reference the state object?

Reply:
Good question! KualiCo added the GeneratePdpGlpeState object with the 1/4 upgrade and this has methods that will get the doc type for the given disbursement type. Different maps are used for the different cases like stop, stale cancelled, reissued, etc. This method was updated to follow base code changes.

Reply:
Thanks, I shall merge